src/main/java/com/indeed/jiraactions/Action.java

```diff
@@ -42,6 +44,16 @@
     Map<CustomFieldDefinition, CustomFieldValue> getCustomFieldValues();
     Set<Link> getLinks();
 
+    // Jiraissues Fields - Building these fields alongside the jiraactions fields does not affect the jiraactions TSV unless they are added into its headers through TsvFileWriter
```

Comment:
I see this comment, but I'm concerned about the underlying architectural question.